src/controllers/AbstractController.ts

```diff
@@ -196,6 +217,10 @@ export default abstract class AbstractController<T extends AbstractService> {
 		);
 	}
 
+	private verifyInt(num: Number): boolean {
+		return Number.isInteger(num) && num > 0;
+	}
```

Comment:
Just a random thought; I wonder how many of these sorts of functions could just be standalone utility functions and not be associated with a class in some way (once something is in a class, it has access to everything under `this` and so in my mind can be a little harder to reason about what they might do, whereas with standlone functions in a utility file you know exactly all of the data they will have access to)

Reply:
Yea I am open to pushing this in our util folder. It's newly created with the `PromiseQueue` being added to sidecar so it would be nice to have these simpler verification helper functions in there.

Reply:
It's a start to definitely clean things up.

Reply:
Okay cool, so the verifyInt has all been removed and added to its own integers util folder.